fragment_internal_test.go

```diff
@@ -2417,6 +2417,11 @@ func (f *fragment) Clean(t testing.TB) {
 	}
 }
 
+// importRoaringT calls importRoaring with context.Background() for convenience
+func (f *fragment) importRoaringT(data []byte, clear bool) error {
```

Comment:
this seems reasonable-ish, but i might sorta prefer ... hmm. i have no good ideas. maybe something like "allow a nil context and default to context.Background"? but that doesn't really feel better.